language/de/install.php

```diff
@@ -356,15 +357,38 @@
 
 		<br />
 
+		<h1>Wie du ein Update deiner Installation mit dem „Komplettpaket“ durchführst</h1>
+
+		<p>Es wird empfohlen deine Installation mit Hilfe des Komplettpakets zu aktualisieren. Wenn in deiner Installation die phpBB-Hauptdateien verändert wurden, solltest du das „Automatische-Update-Paket“ verwenden, um diese Veränderung nicht zu verlieren. Du kannst deine Installation auch mit den in der INSTALL.html beschriebenen Methoden aktualisieren. Zum Aktualisieren von phpBB mit Hilfe des Komplettpakets musst du folgende Schritte ausführen:</p>
+
```

Comment:
Einerseits wird empfohlen das Komplettpaket zu verwenden, gleich im nächsten Satz der Autoupdater. Finde ich ungünstig. Sollten wir wenigstens etwas abgrenzen wie "Wenn allerdings in deiner Installation [...] solltest du stattdessen ..."